docs/attributes.md

```diff
@@ -32,6 +32,7 @@ Configurable (with defaults)
 * `node['splunk']['config']['host']` - Hostname to configure the Splunk instance to report as. (EC2 Instance ID or Fully Qualified Domain Name)
 * `node['splunk']['config']['roles']` - Data bag item used to configure roles (`nil` - roles not managed by chef)
 * `node['splunk']['config']['licenses']` - Data bag item that the license server recipe uses as the source of truth for the license data (`cerner_splunk/licenses`)
+* `node['splunk']['config']['license-pool']` - Data bag item used to configure license pools (`cerner_splunk/license-pool`)
```

Comment:
I don't see where the default value is coming from.  While we're at it, the splunk/config/licenses attribute above doesn't appear to have a default value either. Might as well go ahead and fix the doc for both of them.